refactor(permissions): Note hiding of commands when setting default member permissions (#1617)
discordjs/guide: guide/slash-commands/permissions.md

## Member permissions

> You can use <DocsLink section="builders" path="SlashCommandBuilder:Class#setDefaultMemberPermissions" type="method" /> to set the default permissions required for a member to run the command. Setting it to `0` will prohibit anyone in a guild from using the command unless a specific overwrite is configured or the user has the Administrator permission flag. This also controls the command's visibility, ensuring that only users with the required permissions can access it.

Reviewer: Seems adding this word basically states the same intention. What do you think?

```suggestion
You can use <DocsLink section="builders" path="SlashCommandBuilder:Class#setDefaultMemberPermissions" type="method" /> to set the default permissions required for a member to run the command. Setting it to `0` will hide and prohibit anyone in a guild from using the command unless a specific overwrite is configured or the user has the Administrator permission flag.
```

Author: It does look cleaner, but i'm not sure if the message is the same. I tried to emphasize it just in case the user following the guide is a beginner and needs the extra information, but yeah, it looks fine to me.
